JenkinsFile_build_doc.groovy

```diff
@@ -67,8 +70,17 @@ timeout(time: 6, unit: 'HOURS') {
                             setBuildStatus(OPENJ9_REPO, MERGE_COMMIT, 'PENDING', 'In Progress')
                             currentBuild.description = "${MERGE_COMMIT.take(8)} - ${COMMIT_MSG}"
                         }
-
-                        sh 'mkdocs build -v'
+						// Need to insert website banners to identify the documentation
+						
+						if (params.ghprbPullId) {
+							// Staging site banner
+							sh "sed -i 's|{% block hero %}|<!--Staging site notice --><div class="md-container"><div style="margin:-5rem 0 -5rem 0; background: linear-gradient(white, #69c1bd, white); color:white; text-align:center; font-size:1.5rem; font-weight:bold; text-shadow: 2px 2px 4px #000000;"><p style="padding:2rem 0 2rem 0;"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> CAUTION: This site is for reviewing draft documentation. For published content, visit <a style="color:#af6e3d; text-shadow:none; padding-left:1rem;" href="http://www.eclipse.org/openj9/docs/index.html">www.eclipse.org/openj9/docs</a></p></div>{% block hero %}|' theme/base.html"
+						}
+                        else {
+							// Ghpages site banner
+							sh "sed -i 's|{% block hero %}|<!--Ghpages site notice --><div class="md-container"><div style="margin:-5rem 0 -5rem 0; background: linear-gradient(white, #ffa02e, white); color:white; text-align:center; font-size:1.5rem; font-weight:bold; text-shadow: 2px 2px 4px #000000;"><p style="padding:2rem 0 2rem 0;"><i class="fa fa-cogs" aria-hidden="true"></i> CAUTION: This site hosts draft documentation for the next release. For published content of the latest release, visit <a style="color:#af6e3d; text-shadow:none; padding-left:1rem;" href="http://www.eclipse.org/openj9/docs/index.html">www.eclipse.org/openj9/docs</a></p></div>{% block hero %}|' theme/base.html"
```

Comment:
I'm no website expert by any stretch but I wonder if these banners could be stored in file(s) in the repo and then called upon at this step rather than hardcoding them here?

Reply:
I did wonder about this too. Let's get it working and then decide if the strings should live alongside the base.html in the "themes" directory